hdk-rust/src/api.rs

```diff
@@ -159,12 +159,48 @@ pub fn debug(msg: &str) -> ZomeApiResult<()> {
 
 /// Not Yet Available
```

Comment:
Rust doc? @Connoropolous?

Reply:
Just a simple thing like `call() allows you to call a function in another zome.`  should suffice